Meziantou.Xunit.v3.ParallelTestFramework.Tests/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "diagnosticMessages": true,
+  "preEnumerateTheories": true
```

Comment:
`preEnumerateTheories` is mandatory with ParallelTestFramework.
With Microsoft Testing Platform, the default has been switching in xunit to default this to off, so this will be a common gotcha for consumers of this library. This had me scratching my head for a while.

Reply:
It can mean things work in `dotnet test`, but not with `dotnet run`

Reply:
Done

Reply:
Oh wait, I've got the wording in the README backwards, let me reword it...